diff --git a/src/click/shell_completion.py b/src/click/shell_completion.py
index 8a2668b..125016f 100644
--- a/src/click/shell_completion.py
+++ b/src/click/shell_completion.py
@@ -724,8 +724,8 @@ def _resolve_context(
                     with cmd.make_context(
                         name, args, parent=ctx, resilient_parsing=True
                     ) as sub_ctx:
-                        ctx = sub_ctx
                         args = ctx._protected_args + ctx.args
+                        ctx = sub_ctx
                 else:
                     sub_ctx = ctx
 
@@ -743,8 +743,8 @@ def _resolve_context(
                             allow_interspersed_args=False,
                             resilient_parsing=True,
                         ) as sub_sub_ctx:
-                            sub_ctx = sub_sub_ctx
                             args = sub_ctx.args
+                            sub_ctx = sub_sub_ctx
 
                     ctx = sub_ctx
                     args = [*sub_ctx._protected_args, *sub_ctx.args]
